tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/BRunUtil.java

```diff
@@ -1334,10 +1342,13 @@ public static ExitDetails run(CompileResult compileResult, String... args) {
         try {
             final List<String> actualArgs = new ArrayList<>();
             actualArgs.add(0, "java");
-            actualArgs.add(1, "-cp");
+            for (int indexNum = 0; indexNum < javaOpts.size(); indexNum++) {
+                actualArgs.add(indexNum + 1, javaOpts.get(indexNum));
+            }
```

Comment:
can't we call this parameter `index` instead of `indexNum`?
Shall we initialize this parameter outside of the loop. 
1. Initiate index outside of the loop as `int index = 0`
2. Use it throughout when accessing indices of `actualArgs` by incrementing it
3. change the check for the loop as `index <= javaOpts.size()`. We can very well use a while loop here

Reply:
I will look into this.